core/cog-viewport.h

```diff
@@ -29,6 +29,14 @@ struct _CogViewportClass {
     void *_padding[8];
 };
 
+#define COG_TYPE_VIEWPORT_IMPL (cog_viewport_get_impl_type())
+
+COG_API
+GType cog_viewport_get_impl_type(void);
+
+COG_API
+CogViewport *cog_viewport_new();
```

Comment:
Missing `void` in argument list:

```suggestion
CogViewport *cog_viewport_new(void);
```